frontend/src/metabase/public/components/EmbedFrame/EmbedFrame.tsx

```diff
@@ -154,14 +158,17 @@ export const EmbedFrame = ({
     !!dashboard && isParametersWidgetContainersSticky(visibleParameters.length);
 
   const saveAsPDF = async () => {
-    const cardNodeSelector = `#${DASHBOARD_PDF_EXPORT_ROOT_ID}`;
-    await saveDashboardPdf(
-      cardNodeSelector,
-      name ?? t`Exported dashboard`,
-    ).then(() => {
-      // TODO: tracking
-      // trackExportDashboardToPDF(dashboard.id);
+    const dashboardAccessedVia = match(dashboard?.id)
+      .when(isJWT, () => "static-embed" as const)
+      .when(isUuid, () => "public-link" as const)
+      .otherwise(() => "sdk-embed" as const);
```

Comment:
this is EmbedFrame, so if the id is not a jwt or a uuid, it means it's a normal id used in the sdk.
We don't have tests on the sdk, but there's a test that checks that normal dashboards send 'internal'